packages/@react-types/shared/src/collections.d.ts

```diff
@@ -69,9 +69,9 @@ export interface CollectionStateBase<T, C extends Collection<Node<T>> = Collecti
 
 export interface Expandable {
   /** The currently expanded keys in the collection (controlled). */
-  expandedKeys?: Iterable<Key>,
+  expandedKeys?: 'all' | Iterable<Key>,
   /** The initial expanded keys in the collection (uncontrolled). */
-  defaultExpandedKeys?: Iterable<Key>,
+  defaultExpandedKeys?: 'all' | Iterable<Key>,
   /** Handler that is called when items are expanded or collapsed. */
   onExpandedChange?: (keys: Set<Key>) => any
```

Comment:
Should we mark these as alpha as well? Not really being used except in other alpha packages

Reply:
if they aren't being used except for in other alpha packages i'd be inclined to mark them as alpha as well

Reply:
Addressed by duplicating the Expandable props in SpectrumTableProps